### ISSUE TITLE
spaceAroundParens removes space between closure type and capture list.

### ISSUE THREAD

**Issue by rerelease** [nicklockwood/SwiftFormat#141]
`spaceAroundParens` changes

```swift
service.doAsyncThing { [weak self] () -> Void in
```
to
```swift
service.doAsyncThing { [weak self]() -> Void in
```

Is this intentional? It compiles but does not look quite right IMO.

**Comment by nicklockwood** [comment]
@rerelease it's not exactly intentional. It's because SwiftFormat doesn't distinguish between

    let foo = [...]()

Where the space *should* be removed, and

   { [...] () -> Void in ... }

Where it arguably shouldn't. I'll add a special-case.

**Comment by nicklockwood** [comment]
Fixed in 0.28.6.
